content/docs/csidriver/installation/test/powermax.md

```diff
@@ -70,3 +70,89 @@ Use this procedure to perform a volume expansion test.
   - Writes some data to the PVC
   - After that, it calculates the checksum of the written data, expands the PVC, and then recalculates the checksum
   - Cleans up all the resources that were created as part of the test
+
+## Consuming existing volumes with static provisioning
```

Comment:
Does this procedure work with any masking view name ? Or will the driver figure out and will create the necessary objects.

The request was from a customer who wanted to pre-provision a LUN.

Reply:
hi @coulof  , the driver will create the necessary objects like SG , Host id and MV .

Reply:
So a customer is forbidden to create these objects right ?
If so let's add a note to say the same.

Reply:
Addressed